api/api.py

```diff
@@ -586,12 +586,17 @@ def enable_feature(feature_name: str) -> None:
 def local_resource(name: str, cmd: str, deps: Union[str, List[str]] = None,
                    trigger_mode: TriggerMode = TRIGGER_MODE_AUTO,
                    resource_deps: List[str] = [], ignore: Union[str, List[str]] = [],
-                   auto_init: bool=True) -> None:
-  """Configures `cmd` to run on the *host* machine (not in a remote cluster).
+                   auto_init: bool=True, serve_cmd: str = "") -> None:
+  """Configures one or more commands to run on the *host* machine (not in a remote cluster).
 
-  If `deps` is set then `cmd` is run whenever one of the files specified changes.
+  By default, local_resources are updated on ``tilt up`` and whenever any of their ``deps`` change.
```

Comment:
sigh, "updated" feels unclear to me here :-/

mayyybe: "local resources are update (i.e. executed) on `tilt up`..."? or is that worse? :-/

Reply:
Thanks for persisting on this annoying issue that I despaired of tackling.

I've adjusted it and it's still not great, but I think it's much better. I'm gonna go ahead and merge, but am happy to edit / discuss further if you have more thoughts.